<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Telemetry</class>
 <widget class="QWidget" name="Telemetry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="acc_engineer::ui::telemetry_tyre" name="FrontLeftTyre">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="acc_engineer::ui::telemetry_tyre" name="FrontRightTyre">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="acc_engineer::ui::telemetry_tyre" name="RearLeftTyre">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="acc_engineer::ui::telemetry_tyre" name="RearRightTyre">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>320</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>acc_engineer::ui::telemetry_tyre</class>
   <extends>QFrame</extends>
   <header location="global">ui/telemetry_tyre.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
